### ISSUE TITLE
CMS: Add duplicate buttons in ACL editor

### ISSUE THREAD

**Issue by ivanslabbert** [bccsa/luminary#141]
Add duplicate buttons as indicated below
![telegram-cloud-photo-size-4-5875097253692162645-y](https://github.com/bccsa/luminary/assets/13463323/b61037fa-049a-4ed6-9d10-0e2e227606c3)

**Comment by belljohan3** [comment]
Instead of a button, i added an Icon which hide when are changes and show are not. @laurensgroeneveld Helps me to write tests for that. 

Now i have to add a duplicate icon or button on the ACL.

**Comment by ivanslabbert** [comment]
@belljohan3 please have a look at the colour of the duplicate button? It feels a bit different from the other icons. Should it perhaps be the same colour as the "group" icon that is displayed to the left of the group name?

<img width="1024" alt="image" src="https://github.com/bccsa/luminary/assets/13463323/300a0698-9869-4f37-955e-92ef4c7c7688">

**Comment by laurensgroeneveld** [comment]
@ivanslabbert I agree, it was kind of deliberate to focus a bit more on the functionality and leave nicer styling for a follow-up PR. I created a small one to add a specific variant for this button that also removes the padding, which is a thing we ran into when adding the duplicate button for the individual ACLs. See https://github.com/bccsa/luminary/pull/144

Would love to hear what you think @belljohan3.

<img width="1329" alt="image" src="https://github.com/bccsa/luminary/assets/39056299/9f5dd1ab-b6d2-4610-a073-02c92dd1208f">
